ivre/db/mongo.py

```diff
@@ -1142,6 +1142,32 @@ def searchscreenshot(port=None, protocol='tcp', service=None, neg=False):
                            'screenshot': {'$exists': not neg}}
         }}
 
+    @staticmethod
+    def searchcpe(value=None, type=None, vendor=None, product=None,
```

Comment:
I'd really prefer another name for `type`, such as `cpetype` or `cpe_type` for example (even if it is redundant with the method name `searchcpe`), or maybe `type_` if you like this one better (see [PEP8](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles) `single_trailing_underscore_`).